apps/web/src/App.tsx

```diff
@@ -194,6 +194,11 @@ function RequiredAuth({ children }: any) {
     window.location.reload();
   }
 
+  // Logout if token.exp is in the past (expired)
+  if (payload && payload.exp && payload.exp <= Date.now() / 1000) {
+    logout();
```

Comment:
Also, a small suggestion, worth returning here so the rest of the function won't execute it may prevent a flicker of loading the current page.

Reply:
Okay